### PR TITLE
refactor: refact daemon log

### PR DESCRIPTION
**Pull request by rudyfly**
### Ⅰ. Describe what this PR did
Change the log level of all the "Get" method from "Info" to "Debug".
Add the log to show set the disk quota information.

### Ⅱ. Does this pull request fix one issue?

NONE

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
NONE

### Ⅳ. Describe how to verify it
NONE

### Ⅴ. Special notes for reviews

Signed-off-by: Rudy Zhang <rudyflyzhang@gmail.com>

### CODE REVIEW

File: cri/v1alpha1/cri.go [hunk 1]
```diff
@@ -855,7 +855,7 @@ func (c *CriManager) ListContainerStats(ctx context.Context, r *runtime.ListCont
 	for _, container := range containers {
 		cs, err := c.getContainerMetrics(ctx, container)
 		if err != nil {
-			logrus.Errorf("failed to decode metrics of container %q: %v", container.ID, err)
```
Comment (HusterWan) [inline cri/v1alpha1/cri.go]: decode metrics log level keep as `Error`

Comment (fuweid) [reply cri/v1alpha1/cri.go]: +1

**Comment by Starnop** [comment]
LGTM. expect the comments above. Just a little tip: `ContainerStats` will not be called by kubernetes automatically.

File: cri/v1alpha1/cri_wrapper.go [hunk 2]
```diff
@@ -94,12 +94,12 @@ func (c *CriWrapper) RemovePodSandbox(ctx context.Context, r *runtime.RemovePodS
 
 // PodSandboxStatus returns the status of the PodSandbox.
 func (c *CriWrapper) PodSandboxStatus(ctx context.Context, r *runtime.PodSandboxStatusRequest) (res *runtime.PodSandboxStatusResponse, err error) {
```
Comment (allencloud) [inline cri/v1alpha1/cri_wrapper.go]: So strange to see using a defer to cover a simple error asserting. 😆

**Pull request merged**